app/controllers/companies_controller.rb

```diff
@@ -13,4 +15,14 @@ def show
     end
   end
 
+  private
+
+  def location_list
```

Comment:
I wonder if we can pull these methods out to the helper to keep the controller skinny.